scripts/trim-low-abund.py

```diff
@@ -483,6 +514,24 @@ def main():
         log_info("Saving k-mer countgraph to {graph}", graph=args.savegraph)
         ct.save(args.savegraph)
 
+    if args.output is not None and args.output.name != 1:
+        base = args.output.name
+    # no explicit name or stdout stream get a default name
+    else:
+        base = 'trim-low-abund'
```

Comment:
I'd suggest something with the script name as well as an ISO 8061 compatible timestamp in the log file name.